draft-ietf-tls-esni.md

```diff
@@ -261,7 +261,10 @@ key_config
 : A `HpkeKeyConfig` structure carrying the configuration information associated
 with the HPKE public key. Note that this structure contains the `config_id`
 field, which applies to the entire ECHConfigContents. Sites MUST NOT publish
-two differet`ECHConfigContents` values with the same `HpkeKeyConfig` value.
+two different `ECHConfigContents` values with the same `HpkeKeyConfig` value.
+The RECOMMENDED technique for choosing `config_id` is to choose a random byte.
+This process should be repeated if this config_id matches that of any existing
+(and valid) ECHConfig.
```

Comment:
```suggestion
This process is repeated if this config_id matches that of any valid ECHConfig, which
could include any ECHConfig that has been recently removed from active use.
```

... or something along those lines.  I suspect that operations will want to avoid collisions over slightly longer timescales.